kit/azure/meshplatform/README.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Azure meshPlatform
+summary: |
+  integrates this platform with meshStack as a meshPlatform to enable self-service for our engineering teams.
+compliance:
```

Comment:
this is not the only capability meshStack adds :-D 

we can consider exporting all the tool2block links from CFMM if we want. 

open q: if i have azure and aws etc. and thus multiple platform modules, will i have this capability implemented multiple times? probably yes